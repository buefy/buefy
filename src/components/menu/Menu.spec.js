import { shallowMount } from '@vue/test-utils'
import BMenu from '@components/menu/Menu'

let wrapper

describe('BMenu', () => {
    beforeEach(() => {
        wrapper = shallowMount(BMenu)
    })

    it('is called', () => {
        expect(wrapper.name()).toBe('BMenu')
        expect(wrapper.isVueInstance()).toBeTruthy()
    })

    it('render correctly', () => {
        expect(wrapper.html()).toMatchSnapshot()
    })
})
