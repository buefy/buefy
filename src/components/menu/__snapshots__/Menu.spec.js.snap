// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BMenu render correctly 1`] = `<div class="menu"></div>`;
